hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/utils/ContainerLogger.java

```diff
@@ -145,6 +147,16 @@ public static void logRecovered(ContainerData containerData) {
     LOG.info(getMessage(containerData));
   }
 
+  /**
+   * Logged when a container is reconciled.
+   *
+   * @param containerData The container that was reconciled on this datanode.
+   * @param oldDataChecksum The old data checksum.
+   */
+  public static void logReconciled(ContainerData containerData, long oldDataChecksum) {
+    LOG.info(getMessage(containerData, "Container reconciled. Old checksum is " + checksumToString(oldDataChecksum)));
```

Comment:
Maybe reuse `      LOG.warn("Container {} reconciled, Checksum updated from {} to {}", containerData.getContainerID(),
`